pldoc/_components/buttons.md

```diff
@@ -93,7 +93,7 @@ info:          Buttons should be used for performing actions within the edX envi
         <button type="button" class="btn-inverse">Inverse</button>
         <button type="button" class="btn-inverse btn-large">Inverse Large</button>
         <button type="button" class="btn-inverse btn-small">Inverse Small</button>
-        <button type="button" class="btn-inverse" disabled>Inverse Disabled</button>
+        <button type="button" class="btn" disabled>Base Disabled</button>
```

Comment:
I think the problem with replacing the class in this example is that it is still possible to have an inverse button that is disabled and get the old style. Shouldn't we tweak the inverse disabled button styling?

I also think using the base disabled style here sticks out like a sore thumb compared to the other inverse buttons next to it.